config/manali/cpe.py

```diff
@@ -0,0 +1,279 @@
+            'handlers': [
+                {
+                    'type': 'file',
+                    'name': 'reframe.log',
+                    'level': 'debug2',
+                    'format': '[%(asctime)s] %(levelname)s: %(check_info)s: %(message)s',   # noqa: E501
+                    'append': False
+                },
+                {
+                    'type': 'stream',
+                    'name': 'stdout',
+                    'level': 'info',
+                    'format': '%(message)s'
+                },
+                {
+                    'type': 'file',
+                    'name': 'reframe.out',
+                    'level': 'info',
+                    'format': '%(message)s',
+                    'append': False
+                }
+            ],
+            'handlers_perflog': [
+                {
+                    'type': 'filelog',
+                    'prefix': '%(check_system)s/%(check_partition)s',
+                    'level': 'info',
+                    'format': '%(check_job_completion_time)s|reframe %(version)s|%(check_info)s|jobid=%(check_jobid)s|num_tasks=%(check_num_tasks)s|%(check_perf_var)s=%(check_perf_value)s|ref=%(check_perf_ref)s (l=%(check_perf_lower_thres)s, u=%(check_perf_upper_thres)s)|%(check_perf_unit)s',   # noqa: E501
+                    'datefmt': '%FT%T%:z',
+                    'append': True
+                },
+#                 {
+#                     'type': 'httpjson',
+#                     'url': 'http://httpjson-server:12345/rfm',
+#                     'level': 'info',
+#                     'extras': {
+#                         'facility': 'reframe',
+#                         'data-version': '1.0',
+#                     }
+#                 }
+            ]
+        }
+    ],
+    'modes': [
+        {
+            'name': 'maintenance',
+            'options': [
+                '--unload-module=reframe',
+                '--exec-policy=async',
+                '--strict',
+                '--output=$APPS/UES/$USER/regression/maintenance',
+                '--perflogdir=$APPS/UES/$USER/regression/maintenance/logs',
+                '--stage=$SCRATCH/regression/maintenance/stage',
+                '--report-file=$APPS/UES/$USER/regression/maintenance/reports/maint_report_{sessionid}.json',
+                '-Jreservation=maintenance',
+                '--save-log-files',
+                '--tag=maintenance',
+                '--timestamp=%F_%H-%M-%S'
+            ]
+        },
+        {
+            'name': 'production',
+            'options': [
+                '--unload-module=reframe',
+                '--exec-policy=async',
+                '--strict',
+                '--output=$APPS/UES/$USER/regression/production',
+                '--perflogdir=$APPS/UES/$USER/regression/production/logs',
+                '--stage=$SCRATCH/regression/production/stage',
+                '--report-file=$APPS/UES/$USER/regression/production/reports/prod_report_{sessionid}.json',
```

Comment:
Does it make sense to keep these modes here? Since manali doesn't have `$APPS`?

Reply:
https://github.com/eth-cscs/cscs-reframe-tests/pull/33/commits/bfc51f8185cbfcfcadf3f52e58e5551be55da65c